cypress/e2e/add-liquidity.test.ts

```diff
@@ -52,4 +52,51 @@ describe('Add Liquidity', () => {
       cy.get('#add-liquidity-selected-fee .selected-fee-percentage').should('contain.text', '40% select')
     })
   })
+
+  it('disables increment and decrement until initial prices are inputted', () => {
+    // ETH / BITCOIN pool (0.05% tier not created)
```

Comment:
will this test start failing if that tier is created on mainnet?

Reply:
You're correct that it will start failing if the pool becomes created on mainnet. I'm expanding and hardening the test suite for the entire LP flow in a follow-up commit. Will address this.